### DISCUSSION TITLE
How do you change Export method?

### DISCUSSION THREAD

**Discussion by OscarW99** [iSoron/uhabits#1407] [Help & FAQ]
I want to export my data via email. I have previously exported my data to WhatsApp. I am now unable to export my data in ways other than WhatsApp because I selected the 'use always' option when I first exported to WhatsApp. Now the app takes me straight there when I click export. Is there an option to reset the export settings or change to another method?

**Comment by LordJon** [comment]
As far as I'm aware this is something done on OS level (Android) not on app level.
This will probably help you fix the problem: https://www.androidauthority.com/change-default-apps-android-2-633572/
